lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -206,64 +215,65 @@ defmodule ExUnit.DocTestTest do
 
     ExUnit.configure(seed: 0, colors: [enabled: false])
     output = capture_io(fn -> ExUnit.run end)
+             |> String.replace(~r/exs:\d+/, "exs:x") # Ignore line numbers
```

Comment:
How about replacing the line number with `<line number>` instead of `x`? It'll make the later assertions clearer I think.